### PR TITLE
add: `monero-gui-git`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

**Comment by Henryws** [comment]
CI fails

File: packages/monero-gui-git/monero-gui-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    git ls-remote "${url}" master | cut -f1 | cut -c1-8
+}
+version="$(pkgver)"
+
+prepare() {
+    true
+}
+
+build() {
+    make devmode -j"$(nproc)"
```
Comment (Henryws) [inline packages/monero-gui-git/monero-gui-git.pacscript:20]: Is `devmode` required?

Comment (WRM-42) [reply packages/monero-gui-git/monero-gui-git.pacscript:20]: Yes, otherwise it won't compile

**Comment by Henryws** [comment]
So is devmode the compiled release version?

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by WRM-42** [comment]
It uses the latest git commit

**Comment by Henryws** [comment]
Yes I know, I meant what does devmode do? Does it produce a dev binary, or a release binary?

**Comment by Henryws** [comment]
Because it appears that you can replace `devmode` with `release`.

File: packages/monero-gui-git/monero-gui-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
```
Comment (Henryws) [inline packages/monero-gui-git/monero-gui-git.pacscript:21]:
```suggestion
    make release -j"$(nproc)"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by WRM-42** [comment]
> Yes I know, I meant what does devmode do? Does it produce a dev binary, or a release binary?

It produces a dev binary: https://github.com/monero-project/monero#on-linux-and-macos

**Comment by Henryws** [comment]
Ok then replace `devmode` with `release`.

**Comment by WRM-42** [comment]
Doesn't work

cmake

**Comment by Henryws** [comment]
Then update the submodules as it instructs.

**Comment by Henryws** [comment]
CI fails again

**Comment by cat-master21** [comment]
`make install` needs to be replaced by `make release` https://github.com/monero-project/monero-gui#building-on-linux

**Pull request closed without merging**